frontend/web/components/metadata/MetadataPage.tsx

```diff
@@ -0,0 +1,351 @@
+  const createMetadata = () => {
+    openModal(
+      `Create Metadata`,
+      <CreateMetadata
+        onComplete={metadataCreatedToast}
+        organisationId={organisationId}
+      />,
+      'side-modal create-feature-modal',
+    )
+  }
+
+  const editMetadata = (id: string, contentTypeList: MetadataModelField[]) => {
+    openModal(
+      `Edit Metadata`,
+      <CreateMetadata
+        isEdit={true}
+        metadataModelFieldList={contentTypeList}
+        id={id}
+        onComplete={() => {
+          toast('Metadata Updated')
+        }}
+        projectId={projectId}
+        organisationId={organisationId}
+      />,
+      'side-modal create-feature-modal',
+    )
+  }
+
+  const _deleteMetadata = (id: string, name: string) => {
+    openConfirm({
+      body: (
+        <div>
+          {'Are you sure you want to delete '}
+          <strong>{name}</strong>
+          {' metadata?'}
+        </div>
+      ),
+      destructive: true,
+      onYes: () => deleteMetadata({ id }),
+      title: 'Delete Metadata',
+      yesText: 'Confirm',
+    })
+  }
+
+  return (
+    <div>
+      <Row space className='mt-4'>
+        <h5 className='m-b-0'>Metadata</h5>
+        <Button onClick={() => createMetadata()}>{'Create Metadata'}</Button>
+      </Row>
+      <p className='fs-small lh-sm'>Add metadata to your entities</p>
+      <FormGroup className='mt-4'>
+        <PanelSearch
+          id='webhook-list'
+          items={mergeMetadata}
+          header={
+            <Row className='table-header'>
+              <Flex className='table-column px-3'>Name</Flex>
+              <div className='table-column' style={{ width: metadataWidth[0] }}>
+                Environment
+              </div>
+              <div className='table-column' style={{ width: metadataWidth[1] }}>
+                Segment
+              </div>
+              <div className='table-column' style={{ width: metadataWidth[2] }}>
+                Feature
+              </div>
+              <div className='table-column' style={{ width: metadataWidth[3] }}>
+                Remove
+              </div>
+            </Row>
+          }
+          renderRow={(metadata: MergeMetadata) => (
+            <Row
+              space
+              className='list-item clickable cursor-pointer'
+              key={metadata.id}
+              onClick={() => {
+                editMetadata(`${metadata.id}`, metadata.content_type_fields)
+              }}
+            >
+              <Flex className='table-column px-3'>
+                <div className='font-weight-medium mb-1'>{metadata.name}</div>
+              </Flex>
+              <div
+                className='table-column'
+                style={{
+                  alignItems: 'center',
+                  display: 'flex',
+                  width: '185px',
+                }}
+              >
+                <div className='table-column d-flex text-center'>
+                  {metadata.content_type_fields.find(
```

Comment:
Can we pull these out into named variables? Hard to follow the markup

Reply:
With the new changes in the UI, this code will no longer exist.